File: share-code/bai-tap-lap-trinh/cpp/43.cpp
#include <iostream>
#include<iomanip>
using namespace std;

int main()
{
    int scoreOne   = 6;
    int scoreTwo   = 9;
    int scoreThree = 5;
    int maxOneTwo  = max(scoreOne, scoreTwo);
    int maxNumber  = max(maxOneTwo, scoreThree);
    double avg = (double)(scoreOne + scoreTwo + scoreThree) / 3;

    cout << "Max " << maxNumber << endl;
    cout << "avg " << setprecision(2) << avg << endl;

}

File: share-code/bai-tap-lap-trinh/python/basic/43.py
score_one = int(input("Nhập điểm môn thứ nhất: "))
score_two = int(input("Nhập điểm môn thứ hai: "))
score_three = int(input("Nhập điểm môn thứ ba: "))

max_score = max(score_one, score_two, score_three)

average_score = (score_one + score_two + score_three) / 3

rounded_average_score = round(average_score, 1)

print(f"Max: {max_score}")
print(f"Avg: {rounded_average_score}")